web/client/plugins/Toolbar.jsx

```diff
@@ -112,7 +112,7 @@ class Toolbar extends React.Component {
 
     render() {
         const Container = this.props.disableAnimation ? NormalContainer : AnimatedContainer;
-        return (<ToolsContainer id={this.props.id} className={"mapToolbar btn-group-" + this.props.layout}
+        return (<ToolsContainer id={this.props.id} className={`${this.props.layout==="vertical"? 'mapToolbar': 'mapToolbarHorizontal' } btn-group-` + this.props.layout}
```

Comment:
can you instead pass a style object that will override the display prop if it is not vertical?